servicebroker_node/lib/api/api.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function() {
+
+  var fs = require('fs');
+  var path = require('path');
+
+  var settings = JSON.parse(
```

Comment:
We need to assume the catalog.json and any config are provided via external files.
This needs to be injected rather than hardcoded.
We can create a separate chore if you wish.